pkgs/applications/science/biology/subread/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchurl
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "subread";
+  version = "2.0.3";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/subread/subread-${version}/subread-${version}-source.tar.gz";
+    sha256 = "sha256-Vs7zovkU1DJxMGnVwoL0iDHDoezIlDKtVYDKoyKl9Ws=";
+  };
+
+  buildInputs = [
+    zlib
+  ];
+
+  configurePhase = ''
+    cd src
+    cp Makefile.${if stdenv.isLinux then "Linux" else "MacOS"} Makefile
+  '';
+
+  makeFlags = [ "CC_EXEC=cc" ];
+
+  installPhase = ''
+    mkdir $out
+    cp -r ../bin $out
+  '';
+
+  meta = {
+    description = "High-performance read alignment, quantification and mutation discovery";
+    license = lib.licenses.gpl3;
+    maintainers = with lib; [ jbedo ];
+    platforms = with lib.platforms; linux ++ darwin;
+    homepage = "http://subread.sourceforge.net/";
+  };
+
```

Comment:
```suggestion
  meta = with lib; {
    description = "High-performance read alignment, quantification and mutation discovery";
    license = licenses.gpl3;
    maintainers = with maintainers; [ jbedo ];
    platforms = with platforms; linux ++ darwin;
    homepage = "http://subread.sourceforge.net/";
  };
```

Reply:
yeah I was going to do that, fear of `with` now XD